server/matchmaker/search.py

```diff
@@ -123,6 +146,17 @@ def match(self, other: 'Search'):
         :return:
         """
         self._logger.info("Matched %s with %s", self.player, other.player)
+        
+        numgames = self.player.numGames
+        if numgames <= config.NEWBIE_MIN_GAMES:
+            mean, dev = self.unadjusted_rating
+            adjusted_mean = self.adjusted_rating
```

Comment:
That property returns 2 values or None, but result is in one

Reply:
What do you mean ?

Reply:
For now I will leave the @property, because they are used throughout the whole server code and shall we decide to remove them, we would be better doing a huge "clean up" commit that would remove all @property that don't belong.